server/src/types/song.types.ts

```diff
@@ -0,0 +1,21 @@
+import { Types } from 'mongoose';
+import { MongoInjectedType } from './mongo.types';
+
+type SongSchema = {
+  title: string;
+  tempo: Types.Array<string>;
+  originalKey: string;
+  tags: Types.Array<string>;
```

Comment:
is it supposed to be themes or tags? can u verify with CXO? i just wanna double check

Reply:
should be themes